charts/enterprise-metrics/values.yaml

```diff
@@ -593,6 +593,54 @@ ingester:
   extraPorts: []
   env: []
 
+overrides_exporter:
+  replicas: 1
+
+  annotations: {}
+
+  service:
+    annotations: {}
+    labels: {}
+
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+
+  podLabels: {}
+  podAnnotations: {}
+
+  securityContext: {}
+
+  extraArgs: {}
+
+  persistence:
+    subPath:
+
+  livenessProbe:
+    httpGet:
+      path: /ready
+      port: http-metrics
+    initialDelaySeconds: 45
+  readinessProbe:
+    httpGet:
+      path: /ready
+      port: http-metrics
+    initialDelaySeconds: 45
+
+  resources:
```

Comment:
Could we add requests and limits to the `capped-small.yaml` and `capped-large.yaml` files as well. CPU requests need to be rounded up to whole cores and in this case I think the limits should match.

Reply:
Will do, didn't even see the capped values files.

Reply:
Thanks! They only arrived this morning haha